src/functions/helpers.ts

```diff
@@ -508,16 +507,7 @@ function getPredicate(descr: string, isQuery: boolean, locale: Locale): Predicat
     operand = toBoolean(operand);
   }
 
-  const result: Predicate = { operator, operand };
-
-  if (typeof operand === "string") {
-    if (isQuery) {
-      operand += "*";
-    }
-    result.regexp = operandToRegExp(operand);
-  }
-
-  return result;
+  return { operator, operand };
 }
 
 function operandToRegExp(operand: string): RegExp {
```

Comment:
It's probably just because I don't know this code:
- can we have a small docstring to explain this, with some examples ?
- the word `operand` looks misleading to me here. To me it would be `a` or `b` in `a+b`. But as I understand it, it's more like an "expression" or a "regexp-like" string. Am I right?